app/customer.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
+from app.car import Car
+
+
+@dataclass
+class Customer:
+    name: str
+    product_cart: dict[str: int]
+    location: list[int]
+    money: int
+    car: Car
+
+    @classmethod
+    def create_customers(cls, customers: dict) -> list:
```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]